### PR TITLE
Add support for client credentials

### PR DESCRIPTION
**Pull request by z4ce**

### CODE REVIEW

File: network/oauth_client.go [hunk 1]
```diff
@@ -12,19 +12,21 @@ import (
...
+	jar           *cookiejar.Jar
+	context       context.Context
+	username      string
+	password      string
+	target        string
+	timeout       time.Duration
 }
 
-func NewOAuthClient(target, username, password string, insecureSkipVerify bool, includeCookies bool, requestTimeout time.Duration) (OAuthClient, error) {
+func NewOAuthClient(target, username, password string, client_id, client_secret string, insecureSkipVerify bool, includeCookies bool, requestTimeout time.Duration) (OAuthClient, error) {
```
Comment (zachgersh) [inline network/oauth_client.go]: This should be `clientID` and `clientSecret`

**Comment by anwarchk** [comment]
👍

**Comment by z4ce** [comment]
Updated

File: README.md [hunk 2]
```diff
@@ -58,3 +58,23 @@ Commands:
...
+New client secret:  DESIRED_PASSWORD
+Verify new client secret:  DESIRED_PASSWORD
+scope (list):  opsman.admin
+authorized grant types (list):  client_credentials
+authorities (list):  opsman.admin
+access token validity (seconds):  43200
+refresh token validity (seconds):  43200
+redirect uri (list):
+autoapprove (list):
+signup redirect url (url):```
```
Comment (zachgersh) [inline README.md]: triple tick has to be on its own line (or else it renders).

Comment (z4ce) [inline README.md]: Hm, I tested it in GitHub wiki and it rendered right, but I put it on its own line neverthetheless.

**Pull request merged**